src/js/common/browserInfoHelper.js

```diff
@@ -105,4 +105,4 @@ export const getBrowserDoNotTrackStatus = () => {
       windowVar.external.msTrackingProtectionEnabled &&
       windowVar.external.msTrackingProtectionEnabled());
   return isBrowserDoNotTrack ? "true" : "false";
-};
+};
```

Comment:
This file shouldn't have changed - please checkout the same version from master - here is a guide http://nicolasgallagher.com/git-checkout-specific-files-from-another-branch/

Reply:
Cleaned. this up, refer new PR https://github.com/intuit/user-data-for-fraud-prevention/pull/67